### ISSUE TITLE
Collection Creation – Required Fields

### ISSUE THREAD

**Issue by mtribone** [projecthydra/sufia#2106]
When creating a new collection in Sufia 7.0, the text for "* indicates required fields" is still included and the required fields are not marked.

### Expected behavior

Expecting the creation of a collection to align with the creation of works by showing the required metadata with the new required badges.

![screen shot 2016-05-31 at 2 21 13 pm](https://cloud.githubusercontent.com/assets/4163828/15721643/e3741972-2807-11e6-94d6-19ea07710e8b.png)

### Actual behavior

No required badges and old required text

![screen shot 2016-05-31 at 2 18 52 pm](https://cloud.githubusercontent.com/assets/4163828/15685751/1cb6ebec-273b-11e6-828b-afaaf589a25f.png)

### Steps to reproduce the behavior

1. Collections > New Collection
1. Create new collection

### Related work

#2063

**Comment by atz** [comment]
At this point, we don't seem to have a need for the `* required` legend, since we directly include the word `required` with the badge.  Am I to understand you want the asterisk **and** color/text badge? inside or out?  

Then the legend would have to explain "asterisk [required badge] = required" or something distractingly redundant.  I propose we get rid of it while fixing the flagging of required fields here.

**Comment by mtribone** [comment]
@atz Nope. I want the asterisk to be removed just like what you outlined in the previous ticket. The collection creation page should align with the work creation page.

**Comment by mtribone** [comment]
I've updated the screen shot to remove the "* required" and only have the required badge on the title.

**Comment by justcolin** [comment]
When we did this at U-M I seem to remember it could almost—if not entirely—be done in the `simple_form` config files. Do we know what in Sufia is causing it to override CC's new behavior (the required badges)?

**Comment by mjgiarlo** [comment]
@justcolin Here are old-style form overrides:

https://github.com/projecthydra/sufia/blob/5ddf40c98c09af90aa8db66d6e9bba714aa335fe/app/views/collections/_form.html.erb#L3

https://github.com/projecthydra/sufia/blob/48c57aa07c44b75d1d6d3306242aaac65dcac3ee/app/views/curation_concerns/file_sets/_descriptions.html.erb#L4

**Comment by justcolin** [comment]
@mjgiarlo: That makes me feel sad. Unless you or someone else is already working on it I'll throw up a PR to remove those.

**Comment by mjgiarlo** [comment]
@justcolin no one is! have at it, pls, sir. (and if we can close those other two issues this week that you created a while back, that'd be super swell too! :tada:)

**Comment by mjgiarlo** [comment]
@fultonis thanks for nuking the old-style text for required fields. Can you verify if the collection form is using the new-style badges for required fields? Wondering how close this issue is to closing.

**Comment by justcolin** [comment]
@mjgiarlo It is using the new style. There is a bug in CC which fixes the missing required label. About to put in an PR to CC to fix this.

**Comment by mjgiarlo** [comment]
OK, @justcolin. Once that is merged, I'd like to get beta8 cut so we can verify that this is fixed and close this ticket. Thank you, sir.

**Comment by justcolin** [comment]
Thank you! I am also hoping to automate this so we don't have a similar problem in the future.
